### DISCUSSION TITLE
material with textures in different UV map channels

### DISCUSSION THREAD

**Discussion by dzhakhangirov** [google/model-viewer#4400] [Q&A]
Hi! I have a problem with processing material with textures in different UV map channels.
When I upload a model to the editor - https://modelviewer.dev/editor/ 
Everything is shown correctly!
But if I download files using the "Download scene" button and place them on my server, there is a problem with the display - https://arguv.com/ar_viewer/custom/0/bug/UV_map_channels/
3D model - https://arguv.com/ar_viewer/custom/0/bug/UV_map_channels/Sangens_W30S.glb
Can you tell me what the problem might be?

![UV map channels](https://github.com/google/model-viewer/assets/124194927/c8bc8d7b-9a4b-4cf8-80e7-edeffd727030)

**Comment by diegoteran** [comment]
Hi, are you editing the model in the modelviewer editor? If you place the downloaded glb in model viewer editor again, does it show any difference?

**Comment by dzhakhangirov** [comment]
Hi! No, I'm not editing the model in the Model viewer editor.
If I load the model again, there is no difference. It's still the same mistake.
As if the editor uses another library, not this one - https://ajax.googleapis.com/ajax/libs/model-viewer/3.0.1/model-viewer.min.js
I attached a link to the model, you can try it yourself.

**Comment by dzhakhangirov** [comment]
By the way, if you return the old version of the connection with the file below, then everything works fine!
<script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>
Here is the version with the previous script - https://arguv.com/ar_viewer/custom/0/bug/UV_map_channels_old/

**Comment by elalish** [comment]
It looks to me like the problem is that you're using `v3.0.1` when the latest is `v3.2.0`: https://modelviewer.dev/
